elements/pf-tabs/BaseTabs.ts

```diff
@@ -151,26 +149,29 @@ export abstract class BaseTabs extends LitElement {
     BaseTabs.#instances.delete(this);
   }
 
+  async firstUpdated() {
+    this.tabList.addEventListener('scroll', this.#overflow.onScroll.bind(this));
+  }
+
   override render() {
     const { scrollIconSet, scrollIconLeft, scrollIconRight } = this.constructor as typeof BaseTabs;
-
     return html`
-      <div part="container">
-        <div part="tabs-container">${!this.#showScrollButtons ? '' : html`
+      <div part="container" class="${classMap({ overflow: this.#overflow.showScrollButtons })}">
+        <div part="tabs-container">${!this.#overflow.showScrollButtons ? '' : html`
           <button id="previousTab" tabindex="-1"
               aria-label="${this.getAttribute('label-scroll-left') ?? 'Scroll left'}"
-              ?disabled="${!this.#overflowOnLeft}"
-              @click="${this.#scrollLeft}">
+              ?disabled="${!this.#overflow.overflowLeft}"
+              @click="${this.#scrollLeft.bind(this)}">
```

Comment:
shouldn't need to explicitly bind instance methods in lit render templates

Reply:
Ok weird, before not having it there was causing an issue when `this` in the controller.  Came up while @nikkimk and I were pairing.  Removed.